dist/37.index.js

```diff
@@ -0,0 +1,452 @@
+"use strict";
```

Comment:
Do we need to keep transpiled files in the repository?

Reply:
that particular file was because of the node-fetch, which I've reverted to a previous version.  this has been fixed.

Reply:
Do we still need other files in dist? Usually, it's generated during the build step and published but not stored in the repository.